tests/common.h

```diff
@@ -1,7 +1,9 @@
 #pragma once
 #define UNIT_TEST
+#include <core.h>
```

Comment:
We should maybe refactor this name, what do you think? Something like `unit-tests.h` maybe?